app.yaml.template

```diff
@@ -30,9 +30,9 @@ health_check:
 # [START env_variables]
 # Values need to be substituted from the environment
 env_variables:
-    ELASTICSEARCH_URL: ${ES_URL}
+    ELASTICSEARCH_URL: "http://10.132.0.15:9200"
```

Comment:
@mkarmona note that the appengine API and this dev server live in a different google cloud project.. The API lives in the open-targets-eu-dev project, which you should have access to.
To use 10.0.0.x addresses you probably need to spin another instance in the same project.